examples/astro-stores-demo/src/components/store/CategoryPage.tsx

```diff
@@ -75,7 +65,7 @@ export const ProductGridContent = ({
         productsV3.InventoryAvailabilityStatus.PARTIALLY_OUT_OF_STOCK;
 
     return (
-      <SelectedVariant.Root>
+      <SelectedVariant.Root selectedVariantServiceConfig={{}}>
```

Comment:
Maybe we can make all of those optional when the config is empty?
Just to simply use:
```suggestion
      <SelectedVariant.Root>
```

Reply:
Sure